src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+const Big = require('big.js');
```

Comment:
Once you install the proper package from npm you can return to the import statement without getting a linter error.